coderd/audit/request.go

```diff
@@ -397,6 +403,11 @@ func InitRequest[T Auditable](w http.ResponseWriter, p *RequestParams) (*Request
 			}
 		}
 
+		if key, ok := httpmw.APIKeyOptional(p.Request); ok {
+			fields := APIKeyFields(logCtx, p.Log, key)
+			additionalFieldsRaw = mergeAdditionalFields(logCtx, p.Log, additionalFieldsRaw, fields)
+		}
```

Comment:
Is this is a lot of extra data to staple to every audit log? 

If we are trying to debug rbac failures, the raw input is logged on authz failures. Do we need to have scope metadata on all audit log entries?